### ISSUE TITLE
Can not use the combobox scrollbar inside the Tab or Radio Group components 

### ISSUE THREAD

**Issue by banushanv** [tailwindlabs/headlessui#3071]
What package within Headless UI are you using?
@headlessui/vue

What version of that package are you using?
v1.7.19

What browser are you using?
Chrome

Vue version- 3.3.10

![Screenshot 2024-04-01 161105](https://github.com/tailwindlabs/headlessui/assets/35912208/18f6da76-ec9c-408e-9a77-7d4a748785f6)

I can not use the scrollbar inside the headless UI Tab Component, when I click and scroll the scrollbar, the combo box will close. the moment I click inside the scrollbar the Combobox is closed because the Tab component moves focus to the Tab element itself and therefore blurs the Combobox which closes the combobox.

```
<TabGroup>
<TabList class="flex gap-6 p-2 pb-0">
                      <Tab
                        as="template"
                        :key="'1"
                        v-slot="{ selected }"
                      >
                    <button
                    :class="[
                        'py-2.5 px-1 text-left text-sm font-medium leading-5 text-gray-500',
                        'focus:outline-none',
                        selected ? 'border-b-2 border-blue-600 text-blue-600' : ''
                      ]"
                    >
                  <spanTab</span> 
                    </button>
               </Tab>
       <TabPanels class="mt-2">
              <TabPanel
              :key="'2'"
              :class="[
                      'rounded-lg bg-white  shadow-md',
                      'focus:outline-none ring-white ring-opacity-60 ring-offset-2 ring-offset-blue-400 focus:ring-2'
                    ]">    
            <Combobox v-model="selected">
      <div class="relative mt-1">
        <div
          class="relative w-full cursor-default overflow-hidden rounded-lg bg-white text-left shadow-md focus:outline-none focus-visible:ring-2 focus-visible:ring-white/75 focus-visible:ring-offset-2 focus-visible:ring-offset-teal-300 sm:text-sm"
        >
          <ComboboxInput
            class="w-full border-none py-2 pl-3 pr-10 text-sm leading-5 text-gray-900 focus:ring-0"
            :displayValue="(person) => person.name"
            @change="query = $event.target.value"
          />
          <ComboboxButton
            class="absolute inset-y-0 right-0 flex items-center pr-2"
          >
            <ChevronUpDownIcon
              class="h-5 w-5 text-gray-400"
              aria-hidden="true"
            />
          </ComboboxButton>
        </div>
        <TransitionRoot
          leave="transition ease-in duration-100"
          leaveFrom="opacity-100"
          leaveTo="opacity-0"
          @after-leave="query = ''"
        >
          <ComboboxOptions
            class="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black/5 focus:outline-none sm:text-sm"
          >
            <div
              v-if="filteredPeople.length === 0 && query !== ''"
              class="relative cursor-default select-none px-4 py-2 text-gray-700"
            >
              Nothing found.
            </div>

            <ComboboxOption
              v-for="person in filteredPeople"
              as="template"
              :key="person.id"
              :value="person"
              v-slot="{ selected, active }"
            >
              <li
                class="relative cursor-default select-none py-2 pl-10 pr-4"
                :class="{
                  'bg-teal-600 text-white': active,
                  'text-gray-900': !active,
                }"
              >
                <span
                  class="block truncate"
                  :class="{ 'font-medium': selected, 'font-normal': !selected }"
                >
                  {{ person.name }}
                </span>
                <span
                  v-if="selected"
                  class="absolute inset-y-0 left-0 flex items-center pl-3"
                  :class="{ 'text-white': active, 'text-teal-600': !active }"
                >
                  <CheckIcon class="h-5 w-5" aria-hidden="true" />
                </span>
              </li>
            </ComboboxOption>
          </ComboboxOptions>
        </TransitionRoot>
      </div>
    </Combobox>
              </TabPanel>
        </TabPanels>
    </TabGroup>
```

ComboBoxInput is giving issue for closing the scroll bar.

**Comment by Swedish-li** [comment]
Maybe it related to https://github.com/tailwindlabs/headlessui/issues/3033#issuecomment-2029091562

**Comment by RobinMalfait** [comment]
Hey!

This should be fixed by #3104, and will be available in the next release.

You can already try it using:
- `npm install @headlessui/vue@insiders`.
